server/controllers/ssotRetrievalController.js

```diff
@@ -77,3 +78,41 @@ exports.renameBot = async (req, res) => {
         console.error(err);
     }
 };
+
+// GET /createNewBot?userid=78d09f66d2ed466cf20b06f7&botName=NewRobot
+exports.createNewBot = async (req, res) => {
+    try {
+        res.set('Content-Type', 'application/json');
+        const {userid} = req.query;
+        const usableUserId = mongoose.Types.ObjectId(userid);
+        const {botName} = req.query;
+        const nameWithEmptyspace = botName.replace(/\+/g, ' ');
+
+        const ssot = await mongoose.model('SSoT').create({
+            robotMetadata: {
+                starterId: '',
+                robotName: nameWithEmptyspace
+            },
+            elements: []
+        });
+
+        const updatedSsot = await ssot.updateOne({
+            'robotMetadata.robotId': ssot.id
+        }).exec();
+
+        const uao = await mongoose.model('userAccessObject').create({
+            AccessLevel: 'ReadWrite',
+            robotId: ssot.id,
+            userId: usableUserId
+        });
```

Comment:
what are those needed for? they don't seem to be used

Reply:
What are you referring to?

Reply:
those 2 constants, updatedSsot and uao

Reply:
not sure -> @kej-jay ?